cmd/logger/logger.go

```diff
@@ -214,10 +219,96 @@ func logIf(level Level, err error, msg string,
 	}
 }
 
-func errorIf(err error, msg string, data ...interface{}) {
-	logIf(Error, err, msg, data...)
+// FatalIf :
+func FatalIf(err error, msg string, data ...interface{}) {
+	logIf(Fatal, err, msg, data...)
 }
 
-func fatalIf(err error, msg string, data ...interface{}) {
-	logIf(Fatal, err, msg, data...)
+// LogIf :
+func LogIf(ctx context.Context, err error) {
+	if err == nil {
+		return
+	}
+
+	req := GetReqInfo(ctx)
+	if req == nil {
+		req = &ReqInfo{}
+	}
+	tags := make(map[string]string)
+	for _, entry := range req.GetTags() {
+		tags[entry.Key] = entry.Val
+	}
+
+	// Get the cause for the Error
+	message := err.Error()
+	// Get full stack trace
+	trace := getTrace(2)
+	// Output the formatted log message at console
+	var output string
+	if req.API == "" {
+		req.API = "SYSTEM"
+	}
+	if jsonFlag {
+		logJSON, err := json.Marshal(&logEntry{
+			Level:      Error.String(),
+			RemoteHost: req.RemoteHost,
+			RequestID:  req.RequestID,
+			UserAgent:  req.UserAgent,
+			Time:       time.Now().UTC().Format(time.RFC3339Nano),
+			API:        api{Name: req.API, Args: args{Bucket: req.BucketName, Object: req.ObjectName}},
+			Trace:      traceEntry{Message: message, Source: trace, Variables: tags},
+		})
+		if err != nil {
+			panic("json marshal of logEntry failed: " + err.Error())
+		}
+		output = string(logJSON)
+	} else {
+		// Add a sequence number and formatting for each stack trace
+		// No formatting is required for the first entry
+		for i, element := range trace {
+			match, _ := regexp.MatchString("net/http/server.go:([0-9]+):http.HandlerFunc.ServeHTTP\\(\\)", element)
+			trace[i] = fmt.Sprintf("%8v: %s", i+1, element)
+			if match {
+				trace = trace[:i+1]
+				break
+			}
+		}
+		//match, _ := regexp.MatchString("net/http/server.go:([0-9]+):http.HandlerFunc.ServeHTTP\\(\\)", "net/http/server.go:12:http.HandlerFunc.ServeHTTP()")
+		tagString := ""
+		for key, value := range tags {
+			if value != "" {
+				tagString += key + "=" + value + ", "
+			}
+		}
+		apiString := "API: " + req.API + "("
+		if req.BucketName != "" {
+			apiString = apiString + "bucket=" + req.BucketName
+		}
+		if req.ObjectName != "" {
+			apiString = apiString + " ,object=" + req.ObjectName
+		}
+		apiString += ")"
+		timeString := "Time: " + time.Now().Format("15:04:05 MST 01/02/2006")
```

Comment:
create this as a constant `loggerTimeFormat`  and time.Now().UTC()

Reply:
Will save `"15:04:05 MST 01/02/2006"` as constant `loggerTimeFormat`. Was asked to print local time, that is why UTC was removed.

Reply:
done